docs/en/samples/UMAPI SAMPLES.postman_collection.json

```diff
@@ -0,0 +1,1543 @@
+							"",
+							"    pm.sendRequest(JS_RSA_SIGN_LIBRARY_REQUEST, function (err, response) {",
+							"        if (err === null) {",
+							"            pm.environment.set(ENV_JS_RSA_SIGN_LIBRARY, response.text());",
+							"            console.log(\"Successfully loaded JS RSA Sign library.\");",
+							"            getAccessToken();",
+							"        } else {",
+							"            console.log(\"Could not load JS RSA Sign Library.\");",
+							"        }",
+							"    });",
+							"} else {",
+							"    // Already have the JS RSA Sign library loaded, so generate token immediately",
+							"    getAccessToken();",
+							"}",
+							"",
+							"// Tips from: https://github.com/kjur/jsrsasign/issues/199",
+							"// https://github.com/kjur/jsrsasign/wiki/Tutorial-for-JWT-generation",
+							"function getAccessToken() {",
+							"    var navigator = {}; //fake a navigator object for the lib",
+							"    var window = {}; //fake a window object for the lib",
+							"",
+							"    if (!pm.environment.get(ENV_JS_RSA_SIGN_LIBRARY)) {",
+							"        throw new Error(\"Could not load JS RSA Sign Library from https://kjur.github.io/jsrsasign/jsrsasign-latest-all-min.js\");",
+							"    } else if (!pm.collectionVariables.get(\"UMAPI_IMS_ORG\")) {",
+							"        throw new Error(\"UMAPI_IMS_ORG\" + ERROR_MESSAGE);",
+							"    } else if (!pm.collectionVariables.get(\"UMAPI_TECHNICAL_ACCOUNT_ID\")) {",
+							"        throw new Error(\"UMAPI_TECHNICAL_ACCOUNT_ID\" + ERROR_MESSAGE);",
+							"    } else if (!pm.collectionVariables.get(\"UMAPI_IMS\")) {",
+							"        throw new Error(\"UMAPI_IMS\" + ERROR_MESSAGE);",
+							"    } else if (!pm.collectionVariables.get(\"UMAPI_API_KEY\")) {",
+							"        throw new Error(\"UMAPI_API_KEY\" + ERROR_MESSAGE);",
+							"    } else if (!pm.collectionVariables.get(\"UMAPI_META_SCOPE\")) {",
+							"        throw new Error(\"UMAPI_META_SCOPE\" + ERROR_MESSAGE);",
+							"    } else if (!pm.collectionVariables.get(\"UMAPI_PRIVATE_KEY\")) {",
+							"        throw new Error(\"Ensure the Private Key is added to both INITIAL and CURRENT VALUES in the active Postman environment's PRIVATE_KEY variable\");",
+							"    }",
+							"",
+							"    eval(pm.environment.get(ENV_JS_RSA_SIGN_LIBRARY)); //import javascript jsrsasign",
+							"",
+							"    var header = {",
+							"        \"alg\": \"RS256\"",
+							"    };",
+							"",
+							"    var data = {",
+							"        \"exp\": Math.round(87000 + Date.now() / 1000),",
+							"        \"iss\": pm.collectionVariables.get(\"UMAPI_IMS_ORG\"),",
+							"        \"sub\": pm.collectionVariables.get(\"UMAPI_TECHNICAL_ACCOUNT_ID\"),",
+							"        \"aud\": \"https://\" + pm.collectionVariables.get(\"UMAPI_IMS\") + \"/c/\" + pm.collectionVariables.get(\"UMAPI_API_KEY\")",
+							"    };",
+							"    ",
+							"    var meta_scope = \"https://ims-na1.adobelogin.com/s/ent_user_sdk\";",
```

Comment:
This won't work when `UMAPI_IMS` is set to the stage IMS host. We should build the metascope URI with that variable:

```
var meta_scope = "https://"+pm.collectionVariables.get("UMAPI_IMS")+"/s/ent_user_sdk";
```